helpers/getNextConfig.js

```diff
@@ -2,26 +2,23 @@
 
 const { resolve } = require('path')
 
+const { defaultFailBuild } = require('./defaultFailBuild')
 const moize = require('moize')
 
 // Load next.config.js
-const getNextConfig = async function (failBuild = defaultFailBuild) {
+const getNextConfig = async function (failBuild = defaultFailBuild, publishPath = '.') {
```

Comment:
What are your thoughts on using named parameters instead?

```js
const getNextConfig = async function ({ failBuild = defaultFailBuild, publishPath = '.' }) {
```

Like this, you would not need to specify the `failBuild` argument in `getNextDistDir.js`. 
This, in turn, might remove the need to separate `defaultFailBuild()` to its own file.